lib/cloudinary/uploader.rb

```diff
@@ -317,7 +317,7 @@ def self.call_api(action, options)
       params[:signature] = Cloudinary::Utils.api_sign_request(params.reject { |k, v| non_signable.include?(k) }, api_secret)
       params[:api_key]   = api_key
     end
-    timeout = options[:timeout] || Cloudinary.config.timeout || 60
+    timeout = options.fetch(:timeout, Cloudinary.config.to_h.fetch(:timeout, 60))
```

Comment:
```suggestion
    timeout = options.fetch(:timeout) { Cloudinary.config.to_h.fetch(:timeout, 60) }
```

The block is lazily evaluated, unlike when it's simply an argument in parentheses. This saves us the need to create a new hash copy of `Cloudinary.config` when it's not necessary.